source/speech.py

```diff
@@ -436,8 +441,16 @@ def getIndentationSpeech(indentation):
 			res.append(symbol)

```

Comment:
No. We have to process now, because we might have more than 72 spaces 
(18 tabs). I might be able to figure out a way but it'd be messy.


On 6/9/2016 4:50 AM, James Teh wrote:
>
> In source/speech.py 
> <https://github.com/nvaccess/nvda/pull/6057#discussion_r66419846>:
>
> > @@ -436,8 +441,16 @@ def getIndentationSpeech(indentation):
> >  			res.append(symbol)
>
> It just occurred to me that if we're not going to speak, there's no 
> need to call characterProcessing, etc. We could optimise that out. See 
> if you can do this, but don't bother if the code becomes too 
> unreadable as a result (e.g. too many if checks).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/nvaccess/nvda/pull/6057/files/149618ac951d0380dad97bf4bd02ae1f00e451f9#r66419846>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AFGivZ7qBSAmpOk1B5538PjiYo02RO0Cks5qJ-_dgaJpZM4IxqMD>.
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194